src/shared/components/FluxMonacoEditor.tsx

```diff
@@ -166,13 +164,13 @@ const FluxEditorMonaco: FC<Props> = ({
             }}
             editorDidMount={editorDidMount}
           />
-          {isIoxOrg && isInScriptQueryBuilder && (
```

Comment:
can you explain this change to me, why was this previously only for iox orgs and now not?

Reply:
<img width="1419" alt="Screen Shot 2023-06-15 at 1 59 13 PM" src="https://github.com/influxdata/ui/assets/14298407/e2fa68b2-d2dd-4186-bf2e-ce4456f22fa2">

Good question! The `FLUX` shown in the screenshot above was shown in the IOx orgs, but hidden in TSM orgs. Since Flux was the only language in TSM org at that time, it wasn't necessary to show `FLUX`. Since now we are adding InfluxQL to TSM orgs, removing the condition check will display `FLUX` so that users can differentiate which script editor they are using.